textattack/search_methods/population_based_method.py

```diff
@@ -0,0 +1,78 @@
+from abc import ABC, abstractmethod
+
+from textattack.search_methods import SearchMethod
+
+
+class PopulationBasedMethod(SearchMethod, ABC):
```

Comment:
Can you explain more what you mean?

Reply:
Just changing the `PopulationBasedMethod` to `PopulationBasedSearch`. We named greedy/beam `GreedySearch` and `BeamSearch`, rather than `GreedyMethod` and `BeamMethod`.

Reply:
ahh I see I misread your comment